punchcut/src/lib.rs

```diff
@@ -0,0 +1,200 @@
+/*!
+Glyph loading.
+*/
+
+#![forbid(unsafe_code)]
+// TODO: this is temporary-- remove when hinting is added.
+#![allow(dead_code, unused_imports, unused_variables)]
+
+/// Re-export of peniko crate.
+pub use peniko;
+
+mod error;
+mod outline;
+mod scaler;
+
+pub mod source;
+
+/// Representations of fonts and font collections.
+pub mod font {
+    pub use read_fonts::{types::Tag, CollectionRef, FileRef, FontRef, TableProvider};
+}
+
+use font::Tag;
+use source::*;
+
+use core::str::FromStr;
+
+pub use error::*;
+pub use outline::*;
+pub use scaler::*;
```

Comment:
I don't love globs but 🤷

Reply:
The number of symbols is relatively small, so happy to remove the globs :)